tembo-cli/src/cmd/apply.rs

```diff
@@ -41,28 +43,34 @@ const POSTGRESCONF_NAME: &str = "postgres.conf";
 
 /// Deploys a tembo.toml file
 #[derive(Args)]
-pub struct ApplyCommand {}
+pub struct ApplyCommand {
+    #[clap(long, short = 'm')]
+    pub merge: Option<String>,
+}
 
-pub fn execute(verbose: bool) -> Result<(), anyhow::Error> {
+pub fn execute(verbose: bool, _merge_path: Option<String>) -> Result<(), anyhow::Error> {
     info!("Running validation!");
     super::validate::execute(verbose)?;
     info!("Validation completed!");
 
     let env = get_current_context()?;
 
+    let instance_settings = get_instance_settings(_merge_path.clone())?;
```

Comment:
This call feels unnecessary since `get_instance_settings` is already called in the `execute_docker` & `execute_tembo_cloud` functions.